src/hotspot/cpu/x86/sharedRuntime_x86.cpp

```diff
@@ -112,6 +121,11 @@ jne    1b        \n\
     :"=t"(retval)
     :"0"(x), "u"(y)
     :"cc", "ax");
+  } else {
+    jdouble (*addr)(jdouble, jdouble) = (double (*)(double, double))StubRoutines::fmod();
```

Comment:
An assert here will be good to add that StubRoutines::fmod is not null. Or change the if condition above to have that check.

Reply:
Added an assert.